### PR TITLE
feat(dashboard): Price List configurations section

### PR DESCRIPTION
**Pull request by kasperkristensen**
**What**
- Adds section for displaying price list configurations.

Closes CORE-1857

### CODE REVIEW

File: packages/admin-next/dashboard/src/routes/pricing/pricing-detail/components/pricing-configuration-section/pricing-configuration-section.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    <Container className="divide-y p-0">
+      <div className="flex items-center justify-between px-6 py-4">
+        <Heading level="h2">{t("fields.configurations")}</Heading>
+        <ActionMenu
+          groups={[
+            {
+              actions: [
+                {
+                  label: t("actions.edit"),
+                  to: "/configurations/edit",
```
Comment (fPolic) [inline packages/admin-next/dashboard/src/routes/pricing/pricing-detail/components/pricing-configuration-section/pricing-configuration-section.tsx]: nit: wrong link

File: packages/admin-next/dashboard/src/routes/pricing/pricing-detail/components/pricing-configuration-section/pricing-configuration-section.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        <Text leading="compact" size="small" weight="plus">
+          {t("pricing.settings.customerGroupsLabel")}
+        </Text>
+        <Text size="small" className="text-pretty">
+          <span>
+            {firstCustomerGroups.length > 0
+              ? firstCustomerGroups.join(", ")
+              : "-"}
+          </span>
+          {remainingCustomerGroups.length > 0 && (
```
Comment (fPolic) [inline packages/admin-next/dashboard/src/routes/pricing/pricing-detail/components/pricing-configuration-section/pricing-configuration-section.tsx:68]: q: could we use `<ListSummary />` here for consistency

**Review by fPolic — APPROVED** [review]
LGTM

**Comment by olivermrbl** [comment]
Looks like the content overflows the x-axis causing the container to be scrollable:


https://github.com/medusajs/medusa/assets/59018053/d5175ada-e1d1-4354-aa56-5b0b3eb94153

**Pull request merged**